zerver/worker/queue_processors.py

```diff
@@ -527,3 +535,37 @@ def consume(self, event: Mapping[str, Any]) -> None:
                 (stream, recipient, sub) = access_stream_by_id(user_profile, stream_id,
                                                                require_active=False)
                 do_mark_stream_messages_as_read(user_profile, stream)
+
+
+@assign_queue('export_data')
```

Comment:
Let's use the `deferred_work` queue; this seems like a rare enough thing that we don't need several.